include/mockgyver.hrl

```diff
@@ -30,9 +30,14 @@
         ?WITH_MOCKED_SETUP(SetupFun, CleanupFun,
                            ?FOR_ALL_TIMEOUT, ?PER_TC_TIMEOUT)).
 
+-define(WRAP(Type),
+        {'$mock', Type, {?FILE, ?LINE}}).
+
 -define(WRAP(Type, Expr),
         {'$mock', Type, Expr, {?FILE, ?LINE}}).
 
+-define(MOCK_CONTEXT, ?WRAP(m_context)).
+
 -define(MOCK(Expr), ?WRAP(m_init, (Expr))).
```

Comment:
I think this could now be implemented in terms of mockgyver:exec(?MOCK_CONTEXT, Expr). Or, if there is a reason for keeping it, to rename the m_init to something that speaks of one-shot mocks (or something to that effect).

Reply:
Good point. I added a commit that implements it as mockgyver:exec(?MOCK_CONTEXT, Expr), and removes m_init from the parse transform.

Reply:
Great, thanks!